assets/css/screen.css

```diff
@@ -336,24 +336,41 @@ span.post-read-more a:hover {
 }
 
 .article-post {
-    font-family: Merriweather;
+    font-family: Merriweather, Gadugi, Aboriginal Serif, Noto Sans Canadian Aboriginal, serif;
```

Comment:
I did not include Euphemia to prevent the incorrect glyphs for Eastern Cree; instead, Apple users will have Noto Sans Canadian Aboriginal automatically downloaded.